packages/CodingStandard/src/Fixer/ControlStructure/ShortIfReturnFixer.php

```diff
@@ -0,0 +1,216 @@
+<?php
+namespace Symplify\CodingStandard\Fixer\ControlStructure;
+
+use PhpCsFixer\Fixer\DefinedFixerInterface;
+use PhpCsFixer\FixerDefinition\FixerDefinitionInterface;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+final class ShortIfReturnFixer implements DefinedFixerInterface
+{
+
+	public function getDefinition(): FixerDefinitionInterface
+	{
+		return new FixerDefinition(
+			'Short if return should be used.',
+			[
+				new CodeSample('<?php
+if ($value === true) {
+    return true;
+}
+return false;'
+				),
+			]
+		);
+	}
+
+	/**
+	 * Check if the fixer is a candidate for given Tokens collection.
+	 *
+	 * Fixer is a candidate when the collection contains tokens that may be fixed
+	 * during fixer work. This could be considered as some kind of bloom filter.
+	 * When this method returns true then to the Tokens collection may or may not
+	 * need a fixing, but when this method returns false then the Tokens collection
+	 * need no fixing for sure.
+	 *
+	 * @param Tokens $tokens
+	 *
+	 * @return bool
+	 */
+	public function isCandidate(Tokens $tokens)
+	{
+		return $tokens->isAllTokenKindsFound([T_IF, T_RETURN, T_IS_IDENTICAL]);
```

Comment:
I think there might be included `T_FALSE` and `T_TRUE` tokens as well

Reply:
What kind of tokens are these..? I wanted to use them but they didn't work for me and I haven't found 'em here either: http://php.net/manual/en/tokens.php

Reply:
It looks like they don't exists like this, to bad.

Try to look for token id 321:
https://3v4l.org/W3iRj

Should be in list in PHPStorm, when you click on `T_ECHO` e.g., (but it might differ sometimes with real numbers, because PHPStorm's outdated stubs).

Reply:
Btw, this is just a *nice to have*. Not crutial for this Fixer to work.

If it would be a blocker, just skip it

Reply:
token ID 321 is T_INLINE_HTML. Your snippet needs <?php in the beginning ;)
https://3v4l.org/EcoHd
true is still string..

Reply:
I see :) `T_STRING` is it